api/README.md

```diff
@@ -0,0 +1,133 @@
+# To Do API
+
+This is a simple api for  interacting with the boilerplate
```

Comment:
rather add a link to the actual api, instead of documenting it here

Reply:
done